benchmarks/benchmarks_game.toml

```diff
@@ -0,0 +1,25 @@
+[general]
+    # benchmark name
+    name = 'Benchmarks game'
+    # select the most applicable category from the list provided by this repo
+    category = 'PERFORMANCE'
+    # mark your benchmark with custom tags (your own custom tags OR use existing from other projects)
+    tags = ['performance']
+
+[info]
+    # programming language
+    language = 'language agnostic'
```

Comment:
not agnostic...

Reply:
Yes, maybe I didn't phrase it that way. The developers write that these are measurements for more than 24 programming languages.